packages/www/Dockerfile

```diff
@@ -1,12 +1,13 @@
-# To be run from the root of the repository so that we can pick up other packages
+FROM	node:14
 
-FROM node:14
+WORKDIR	/app
+
+COPY	package.json yarn.lock lerna.json	./
+
+RUN	yarn install --frozen-lockfile
+
+COPY	.	.
```

Comment:
This will dump a developer's `node_modules` directory into any locally-built Docker images. Need a `.dockerignore` if we're going this route

Reply:
AFAICT nothing changed in this logic apart from changing `ADD` to `COPY`

Reply:
i added a .dockerignore to keep docker build context small when dev builds images locally